database/src/migrations/20230518125700_update_record_sims_eml_transformation.ts

```diff
@@ -0,0 +1,285 @@
+        pf.proj_n,
+        pf.fund_n,
+        pfs.funds_n,
+        jsonb_build_object(
+          'agencyName',
+          pfs.related_projects_fundings -> 'para',
+          'fundingStartDate',
+          jsonb_path_query_first(
+            pfs.related_projects_fundings,
+            '$.section[*] \\? (@.title == "Funding Start Date").para'
+          ),
+          'fundingEndDate',
+          jsonb_path_query_first(
+            pfs.related_projects_fundings,
+            '$.section[*] \\? (@.title == "Funding End Date").para'
+          )
+        ) funding_object
+      FROM
+        related_projects_funding pf,
+        jsonb_array_elements(funding) WITH ordinality pfs(related_projects_fundings, funds_n)
+    ),
+    projects_funding_arrs AS (
+      SELECT
+        proj_n,
+        jsonb_agg(funding_object) funding_arr
+      FROM
+        projects_fundings
+      GROUP BY
+        proj_n
+    ),
+    related_projects_funding_arrs AS (
+      SELECT
+        proj_n,
+        jsonb_agg(funding_object) funding_arr
+      FROM
+        related_projects_fundings
+      GROUP BY
+        proj_n
+    ),
+    project_objects AS (
+      SELECT
+        jsonb_build_object(
+          'projectId',
+          aps.project -> '@_id',
+          'projectTitle',
+          aps.project -> 'title',
+          'projectOrganizationName',
+          aps.project -> 'personnel' -> 'organizationName',
+          'projectAbstract',
+          jsonb_path_query_first(aps.project, '$.abstract.section'),
+          'taxonomicCoverage',
+          jsonb_path_query_first(
+            aps.project,
+            '$.studyAreaDescription.coverage.taxonomicCoverage'
+          ),
+          'fundingSource',
+          pfas.funding_arr
+        ) project_object
+      FROM
+        projects aps
+        LEFT JOIN projects_funding_arrs pfas ON pfas.proj_n = aps.proj_n
+    ),
+    related_project_objects AS (
+      SELECT
+        jsonb_build_object(
+          'projectId',
+          aps.project -> '@_id',
+          'projectTitle',
+          aps.project -> 'title',
+          'projectOrganizationName',
+          aps.project -> 'personnel' -> 'organizationName',
+          'projectAbstract',
+          jsonb_path_query_first(aps.project, '$.abstract.section'),
+          'taxonomicCoverage',
+          jsonb_path_query_first(
+            aps.project,
+            '$.studyAreaDescription.coverage.taxonomicCoverage'
+          ),
+          'fundingSource',
+          pfas.funding_arr
+        ) project_object
+      FROM
+        related_projects aps
+        LEFT JOIN related_projects_funding_arrs pfas ON pfas.proj_n = aps.proj_n
+    ),
+    project_objects_arr AS (
+      SELECT
+        jsonb_agg(project_object) project_objects_arr
+      FROM
+        project_objects
+    ),
+    related_project_objects_arr AS (
+      SELECT
+        jsonb_agg(project_object) project_objects_arr
+      FROM
+        related_project_objects
+    ),
+    project_type AS (
```

Comment:
this is the only change to the transformation code. Adding new data for the submission metadata table, it is being generated here and added to the transform on line 225